precompiles/dispatch-lockdrop/src/lib.rs

```diff
@@ -57,6 +57,8 @@ pub struct DispatchLockdrop<Runtime, DispatchValidator, DecodeLimit = ConstU32<8
     PhantomData<(Runtime, DispatchValidator, DecodeLimit)>,
 );
 
+type CallLengthLimit = ConstU32<{ 2u32.pow(12) }>;
```

Comment:
Nitpick comment - perhaps it's just me, but I'd find 4 * 1024 more readable 🙃

Reply:
So maybe `4096` directly then